lib/heart_seed/db_seed.rb

```diff
@@ -17,30 +17,49 @@ def self.bulk_insert(source_file: nil, model_class: nil)
+    #
+    # @param file_path [String]
+    # @param model_class [Class] require. extends {ActiveRecord::Base}
+    def self.insert(file_path: nil, model_class: nil)
+      fixtures = HeartSeed::Converter.read_fixture_yml(file_path)
+      model_class.transaction do
+        model_class.delete_all
+        fixtures.each do |fixture|
+          model_class.create(fixture)
+        end
+      end
+    end
+
     # import all seed yaml to table
     #
-    # @param seed_dir [String]
-    # @param tables   [Array<String>,String] table names array or comma separated table names. if empty, import all seed yaml. if not empty, import only these tables.
-    # @param catalogs [Array<String>,String] catalogs names array or comma separated catalog names. if empty, import all seed yaml. if not empty, import only these tables in catalogs.
-    def self.import_all(seed_dir: HeartSeed::Helper.seed_dir, tables: ENV["TABLES"], catalogs: ENV["CATALOGS"])
+    # @param seed_dir    [String]
+    # @param tables      [Array<String>,String] table names array or comma separated table names. if empty, import all seed yaml. if not empty, import only these tables.
+    # @param catalogs    [Array<String>,String] catalogs names array or comma separated catalog names. if empty, import all seed yaml. if not empty, import only these tables in catalogs.
+    # @param insert_mode [String] 'bulk' or other string. if empty or 'bulk', using bulk insert. if not empty and not 'bulk', import with ActiveRecord.
+    def self.import_all(seed_dir: HeartSeed::Helper.seed_dir, tables: ENV["TABLES"], catalogs: ENV["CATALOGS"], insert_mode: ENV["INSERT_MODE"] || "bulk")
       # use tables in catalogs
-      target_tables = parse_arg_catalogs(catalogs)
-      if target_tables.empty?
+      target_table_names = parse_arg_catalogs(catalogs)
+      if target_table_names.empty?
         # use tables
-        target_tables = parse_string_or_array_arg(tables)
+        target_table_names = parse_string_or_array_arg(tables)
       end
 
-      raise "require TABLES or CATALOGS if production" if HeartSeed::Helper.production? && target_tables.empty?
+      raise "require TABLES or CATALOGS if production" if HeartSeed::Helper.production? && target_table_names.empty?
 
       ActiveRecord::Migration.verbose = true
-      Dir.glob(File.join(seed_dir, "*.yml")) do |file|
-        table_name = File.basename(file, '.*')
-        next unless target_table?(table_name, target_tables)
+      Dir.glob(File.join(seed_dir, "*.yml")) do |file_path|
+        table_name = File.basename(file_path, '.*')
+        next unless target_table?(table_name, target_table_names)
 
-        ActiveRecord::Migration.say_with_time("#{file} -> #{table_name}") do
+        ActiveRecord::Migration.say_with_time("#{file_path} -> #{table_name}") do
           begin
             model_class = table_name.classify.constantize
-            bulk_insert(source_file: file, model_class: model_class)
+            if insert_mode == "bulk"
```

Comment:
Magic number `bulk` is scattered in this module ...
I think constant value (ex. `BULK`) is better than magic number.